### DISCUSSION TITLE
metadata in azure cognitive search output for vector_db_lookup_tool?

### DISCUSSION THREAD

**Discussion by eric-sf** [microsoft/promptflow#1077] [Q&A]
Hi,
How can I get metadata in vector_db_lookup_tool for azure cognitive search? It looks like currently, it is returning null. 

 https://microsoft.github.io/promptflow/reference/tools-reference/vector_db_lookup_tool.html

**Comment by 0mza987** [comment]
This discussion is resolved according to https://github.com/microsoft/promptflow/issues/1078
